_posts/2018-05-24-security-session-highlights-from-hkg18.md

```diff
@@ -0,0 +1,29 @@
+---
+author: op-tee
+title: Security Session Highlights from HKG18
+date: 2018-05-24 12:00:00
```

Comment:
If this is merged before 12 then it will not be published as Jekyll will see this as a future post and will only show when the current time is > the date: value